xla/service/gpu/priority_fusion.cc

```diff
@@ -247,13 +247,31 @@ class GpuPriorityFusionQueue {
     return !current_consumers_.empty();
   }
 
+  void UpdatePerformanceModelCache(HloInstruction* producer) {
+    if (!IsFusible(*producer)) {
```

Comment:
There is one tests failing in priority_fusion_test: `CanMergeTritonFusionWithBothProducerAndConsumer`. That's because we try to use the existing cost model for Triton fusion, but kCustom fusion is considered not fusible. It should be enough to add here: `if (!IsFusible(*producer) && !IsGenericTritonFusion(*producer))` as a temporary fix.

Reply:
fixed.